ca/ecdsa_allow_list.go

```diff
@@ -87,19 +86,17 @@ func makeRegIDsMap(regIDs []int64) map[int64]bool {
 }
 
 // NewECDSAAllowListFromFile is exported to allow `boulder-ca` to
-// construct a new `ECDSAAllowList` object and set the initial allow
-// list using the contents of a YAML file. An initial entry count is
```

Comment:
I see you removed from the doccomment "and set the initial allow list using the contents of a YAML file." It seems like the function still does that; is this just a style update (e.g. you decided it was redundant) or is it meant to reflect a functionality change?

Reply:
> Overall this change looks fine, but I can't spot the bug it fixes, which suggests I'm missing something. :-)

It was kind of a tricky bug! In `main` I declare a var `ecdsaAllowList` of type `*ca.ECDSAAllowList`. When I call reloader.New(), with a data callback of `ecdsaAllowList.Update`, it attempts to execute the `ecdsaAllowList.Update` but `ecdsaAllowList` is `nil` 😓.

> I see you removed from the doccomment "and set the initial allow list using the contents of a YAML file." It seems like the function still does that; is this just a style update (e.g. you decided it was redundant) or is it meant to reflect a functionality change?

I adjusted the comment because the update is happening when the `reloader.New()` calls `allowList.Update` instead of `NewECDSAAllowListFromFile` explicitly calling allowList.Update before returning like it was before. I figured that the doc comment for `reloader.New()`, "New loads the filename provided, and calls the callback", had it covered. I'm fine either way though.